src/app/index.ts

```diff
@@ -287,20 +287,35 @@ module.exports = class extends yo {
 
   _configureProject(answerForProjectType, answerForScriptType, answerForHost, answerForName, isManifestProject, isExcelFunctionsProject): void {
     try {
+      const projType = _.toLower(this.options.projectType) || _.toLower(answerForProjectType.projectType)
+
       this.project = {
         folder: this.options.output || answerForName.name || this.options.name,
+        host: answerForHost.host
+          ? answerForHost.host
+          : this.options.host
+          ? this.options.host
+          : jsonData.getHostTemplateNames(projType)[0],
         name: this.options.name || answerForName.name,
-        projectType: _.toLower(this.options.projectType) || _.toLower(answerForProjectType.projectType),
+        projectType: projType,
+        scriptType: answerForScriptType.scriptType
+          ? answerForScriptType.scriptType
+          : this.options.ts
+          ? typescript
+          : this.options.js
+          ? javascript
```

Comment:
Add an indent level here.

Reply:
I'm using the prettier here. So those two spaces are the recommended amount.